starter-code/index.html

```diff
@@ -1,42 +1,77 @@
 <!DOCTYPE html>
 <html lang="en">
+
+<!-- Head -->
+
 <head>
+  <link rel="stylesheet" type="text/css" href="./css/main.css"
   <meta charset="UTF-8">
   <title>Spotify</title>
 </head>
+
+<!-- Body -->
+
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+
+<!-- Navbar -->
+<div class="nav">
+  <img src="./images/spotify-logo.png" alt="Spotify Logo" class="logo">
+  <div class="menuItems">
+    <a href="#">Premium</a>
+    <a href="#">Discover</a>
+    <a href="#">Help</a>
+    <a href="#">Download</a>
+  </div>
+</div>
+
+<!-- Large image background with text -->
+<div class="header">
+  <div class="headerText">
+    <h1>Music for everyone.</h1>
+    <h3>Spotify is now free on mobile, tablet and computer.<br>Listen to the right music, wherever you are.</h3>
+  </div>
+</div>
+
+<!-- What's on Spotify section -->
+<div class="what">
+  <h2>What’s on Spotify?</h2>
+  <div class="underline"></div>
```

Comment:
Clever! ;-) The other way to go about this is using a box and then having the bottom border in format like this: 
```
.someClass{
   bottom-border: 1px solid green;
}
```